lambda/reporting.tf

```diff
@@ -34,8 +35,9 @@ resource "aws_lambda_function" "reporting_lambda_function" {
 
 resource "aws_kms_ciphertext" "environment_vars_reporting" {
   for_each = local.count_reporting == 0 ? {} : {
-    slack_bot_token = var.slack_bot_token,
-    client_secret   = var.keycloak_reporting_client_secret
+    slack_bot_token          = var.slack_bot_token,
+    client_secret            = var.keycloak_reporting_client_secret
+    tdr_reporting_channel_id = var.reporting_slack_channel_id
```

Comment:
Maybe rename this `tdr_reporting_slack_channel_id` to make clears it's a slack channel id